### PR TITLE
Fix broken external link for LDA implementation

### PR DESCRIPTION
**Pull request by ahaya3776**
The old link was broken since Blei moved from Princeton to Columbia, as mentioned in #3141. It seems there was LDA implementation in C, and I fixed the link.

### CODE REVIEW

**Comment by piskvorky** [comment]
Thanks!

I'm not a fan of updating our docs every time someone changes departments though. Can you think of a better way to handle this?

Google shows me https://github.com/blei-lab/lda-c . Looks semi-official… maybe we can link to that?

**Comment by ahaya3776** [comment]
Thanks for the comment, I fixed the link to be more permanent.

File: docs/src/auto_examples/core/sg_execution_times.rst [hunk 1]
```diff
@@ -5,10 +5,10 @@
 
 Computation times
 =================
-**00:03.242** total execution time for **auto_examples_core** files:
+**02:36.161** total execution time for **auto_examples_core** files:
 
 +--------------------------------------------------------------------------------------------------------------+-----------+---------+
-| :ref:`sphx_glr_auto_examples_core_run_corpora_and_vector_spaces.py` (``run_corpora_and_vector_spaces.py``)   | 00:03.242 | 48.2 MB |
+| :ref:`sphx_glr_auto_examples_core_run_corpora_and_vector_spaces.py` (``run_corpora_and_vector_spaces.py``)   | 02:36.161 | 40.1 MB |
```
Comment (piskvorky) [inline docs/src/auto_examples/core/sg_execution_times.rst:11]: This is weird. From 3 seconds to 2.5 minutes? What happened?

Comment (mpenkov) [reply docs/src/auto_examples/core/sg_execution_times.rst]: I'm not sure. I ran the build again on my linux dev machine (the previous one was on MacOS) and the time is more appropriate (5s).

**Comment by mpenkov** [comment]
Merged. Thank you @ahaya3776 !

**Pull request merged**